templates/base.html

```diff
@@ -46,7 +46,7 @@
 
     {{ js('preload') }}
   </head>
-  <body class="html-{{ DIR }} {{ request.APP.short }} moz-header-slim {% block bodyclass %}{% endblock %}"
+  <body class="html-{{ DIR }} {{ request.APP.short }} moz-header-slim {% block bodyclass %}{% endblock %} {% if waffle.flag('restyle') %}restyle{% endif %}"
```

Comment:
The space for the restyle class could be inside the if block so there's not an extra space when it's not present.